scripts/calculateKpi/protocols/tetherV0/parseReferralTag/getTransactionInfo.ts

```diff
@@ -0,0 +1,111 @@
+import {
```

Comment:
this is copied from https://github.com/divvi-xyz/divvi-relayer/blob/main/src/endpoints/submitReferral/getTransactionInfo.ts, and for simplicity i also moved the `TransactionInfo` type here (it is in a types.ts file in the relayer)